gulpfile.js

```diff
@@ -98,6 +98,19 @@ gulp.task('webpack:watch:scss-to-json', (cb) => {
 });
 
 /**
+ * Accessibility test a subset of generated HTML files
+ */
+const { pa11yConcurrent } = require('./tools/tasks/gulp-pa11y');
+
+gulp.task('test:accessibility', () => gulp
+  .src([
+    'dist/public/patterns/**/*.html', // All html
+    '!dist/public/patterns/**/*.markup-only.html', // Except the fragment pages
+    '!dist/public/patterns/**/index.html', // Except the aggregate, auto-generated pages
```

Comment:
Per my comment on #114, I suppose the answer to "how can we test against Drupal", would be an alternate execution approach that queries URLs instead of files?

Reply:
Indeed, the goal for this PR is *just* PL-rendered HTML. But the tools are in and `gulpfile.js` is ready for new entries to test a local/remote Drupal site.

Reply:
Okay, I started to write up a whole thing, but if this is at least open to it, we have the basis to discuss as a follow-up. I'm not convinced this is the right place for Drupal testing, but as long as it's a possible place we can work it out in The Future.